gatsby/gatsbysourceplone.rst

```diff
@@ -0,0 +1,62 @@
+gatsby-source-plone
+===================
+
+With the previous sections on source nodes, retrieving data from ``plone.restapi`` and finally using the search traversal method, we have understood how our source-plugin works at base level.
+
+Cheers! That was amazing work.
```

Comment:
I don't know if we can be so enthusiast in the docs.. @svx ?

Reply:
@cekk good question ! :)
In general it is always tricky, because there is a fine border.

IMHO there is nothing wrong with being motivating, but personally I would rewrite that to be a bit more serious :)
"Cheers" for example may a bit too much :)

But :) For sure do not forget to give your students a "high five" to keep the motivation up during the training in Tokyo ! :)

Reply:
let's talk in singular..right now there are more trainers (2) than students (1) :(

Reply:
I follow these rules of thumb:
- it's OK to praise and reward people for their effort
- it's not OK to praise libraries, tools, and inanimate objects because it often sounds like hype or an judgment, and the objects won't appreciate the praise ;)

There are exceptions to every rule, of course.

Reply:
Thanks for the feedback guys, I'll rewrite that line ;)